### PR TITLE
Merge pull request #96 from rectorphp/controller-test

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: src/Controller/DemoController.php [hunk 1]
```diff
@@ -91,8 +91,6 @@ private function processFormAndReturnRoute(FormInterface $form): RedirectRespons
         $config = $demoFormData->getConfig();
 
         $rectorRun = $this->createRectorRun($config, $demoFormData);
-        $this->rectorRunRepository->save($rectorRun);
-
```
Comment (TomasVotruba) [inline src/Controller/DemoController.php]: This one save() is not needed, right? It's duplicate of the line bellow

Comment (JanMikes) [reply src/Controller/DemoController.php]: Yeah it is no longer required after the refactorings!

Comment (JanMikes) [reply src/Controller/DemoController.php]: Yeah it is no longer required after the refactorings!

Comment (TomasVotruba) [reply src/Controller/DemoController.php]: Thanks for confirmation

**Pull request merged**